### PR TITLE
Accordion dark theme color change

### PR DESCRIPTION
**Pull request by ajstrand**
Associated Issue: #999 

### Summary of Changes

* hovering over accordion in dark theme changes to a color hat fits within the 
color palette decided for the dark color scheme. 

### Testing

* [x] passes `npm test`
* [x] passes `npm run lint`

### Screenshots/Videos (OPTIONAL)
 1. with dark theme enabled(hover color is over the scopes accordion)

2. with light theme enabled(hover color is over the call stack accordion)

@jasonLaster my PR is ready for review. Let me know if there's any changes you would like me to make.

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks for filing this @ajstrand.

Couple comments:
1. I think you checked in the local.json :)
2. I like many of the eslint changes. You're probably using a different version of eslint. Mind filing a separate PR for that stuff and we can discuss
3. I like your header hover fix. You can update this PR to be just that by doing `git push <your remote> <your branch> --no-verify` it's a good trick :)

**Comment by ajstrand** [comment]
@jasonLaster I've updated the PR to revert the commits that didn't relate to the accordion hover color change. Now the only thing that will be merged in is the change I made in `RightSideBar.css`. 

I've made a separate PR to deal with the ESlint issue I had, it is PR #1075. Let me know what further questions/comments you have. 

Sorry that such a simple issue is taking this long to get completed, this is my first OSS contribution experience and frankly the biggest codebase I've seen so far. It took me a while to wrap my head around how everything was laid out.

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 build
 node_modules
 config/local.json
+packages/devtools-config/configs/local.json
```
Comment (jasonLaster) [inline .gitignore]: This doesn't need to be there. Feel free to `rm packages/devtools-config/configs/local.json` this.

Comment (ajstrand) [inline .gitignore]: I've removed that line from the gitignore file. Let me know if there's anything else I need to do.

**Comment by jasonLaster** [comment]
Hmm @ajstrand just took a look and found a couple of things

1. the variable is misspelled
2. the hover color looks a bit strange. 

The current behavior is not bad, but perhaps it can be a bit better. Mind taking another look? If you feel like the current behavior is fine, we can always close this and you can grab a new issue.

**Comment by ajstrand** [comment]
I've fixed the variable misspelling and I chose a different color. Hopefully that one should fit better.

**Comment by jasonLaster** [comment]
Thanks @ajstrand. 

Hmm, I think that that color might be a bit strong.  After looking at this a bit more, i think the current hover color `var(--theme-search-overlays-semitransparent);` is probably a reasonable default. 

Feel free to re-open it if you find a better option. Also, there are lots of other up for grab issues https://github.com/devtools-html/debugger.html/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22 that are more egregious at the moment.

**Comment by ajstrand** [comment]
should i close this pr then?

**Pull request closed without merging**